tigerpath/majors_and_certificates/majors/POL_2018.json

```
@@ -227,20 +461,48 @@
               "name": "The Constitution and the Courts",
               "max_counted": 2,
               "min_needed": 1,
-              "explanation": "",
+              "explanation": null,
               "course_list": [
                 "POL 314",
                 "POL 315",
                 "POL 316",
-                "POL 320"
+                "POL 320",
+                "POL 326"
               ]
             },
             {
               "name": "American Political, Legal, and Constitutional Thought",
               "max_counted": 2,
               "min_needed": 1,
-              "explanation": "",
+              "explanation": null,
+              "course_list": [
+                "WWS 370/POL 308",
+                "POL 314",
+                "POL 315",
+                "POL 316",
+                "POL 319",
+                "POL 321",
+                "POL 328",
+                "POL 332"
+              ]
+            },
+            {
+              "name": "Additional Course",
+              "max_counted": 1,
+              "min_needed": 1,

```

This doesn't work with double-counting, because courses from the others will also count here.
Instead, you could check that there are 5 total courses in the track, which, combined with there being at least 1 in each of the four categories, checks what you want (that there are four in the categories, plus 1 more).
```suggestion
              "name": "5 total courses",
              "max_counted": 1,
              "min_needed": 5,
```